src/renderer/modules/webpack/index.ts

```diff
@@ -24,7 +24,37 @@ import {
 
 // Handlers
 
-let wpRequire: WebpackRequire;
+type GetById = typeof getById;
+type GetByProps = typeof getByProps;
+type GetBySource = typeof getBySource;
+type GetModule = typeof getModule;
+type GetExportsForProps = typeof getExportsForProps;
+type GetFunctionBySource = typeof getFunctionBySource;
+type PatchPlaintext = typeof patchPlaintext;
+type WaitForModule = typeof waitForModule;
+type SourceStrings = typeof sourceStrings;
+
+export namespace webpack {
+  export let wpRequire: WebpackRequire;
+  export let waitForReady: Promise<void>;
+  export let waitForStart: Promise<void>;
+  export let signalStart: () => void;
+  export let getById: GetById;
+  export let getByProps: GetByProps;
+  export let getBySource: GetBySource;
+  export let getModule: GetModule;
+  export let getExportsForProps: GetExportsForProps;
+  export let getFunctionBySource: GetFunctionBySource;
+  export let patchPlaintext: PatchPlaintext;
+  export let waitForModule: WaitForModule;
+  export let sourceStrings: SourceStrings;
+}
```

Comment:
This entire namespace is completely unnecessary and just adds bloat to our API and our code.
```suggestion
```